src/babel-plugin-dynamic-import-amd.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
Seems like this and a few other files are taken out of polymer build? Would it make sense for now to just depend on polymer build, and directly import the relevant modules? Don't want to add all the dependencies? What about factoring them out into another package?

Reply:
I like the idea of factoring things out into smaller packages in theory.  In practice I think we need a plan for how to proceed with that outside the current tools monorepo without proliferating too many micro github repos.

Reply:
Yeah, happy to help brainstorm how that could look IRL sometime.